app/components/filter_component.html.erb

```diff
@@ -26,7 +26,7 @@
               <ul class="moj-filter-tags">
                 <% tags_for_active_filter(active_filter).each do |tag| %>
                   <li>
-                    <%= link_to(tag[:remove_link], class: 'moj-filter__tag', id: "tag-#{tag[:title].parameterize}") do %>
+                    <%= link_to(tag[:remove_link], class: 'app-checkbox-filter__tag', id: "tag-#{tag[:title].parameterize}") do %>
```

Comment:
We should move towards `app-`... not sure this is the best place to start though, even though in theory there is also [no reason not to](https://github.com/DFE-Digital/apply-for-teacher-training/pull/4781/files#diff-5ce4330b11d0313da1f5554adcf07aaf43e13342110eb05ca2d274ef4556cbd7R94)